.travis.yml

```diff
@@ -8,7 +8,7 @@ jdk:
 # Caching per https://docs.travis-ci.com/user/languages/java/
 before_cache:
   - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
-  - rm -f $HOME/.gradle/caches/2.9/plugin-resolution/cache.properties.lock
+  - rm -f $HOME/.gradle/caches/3.5/plugin-resolution/plugin-resolution.lock
```

Comment:
jw, did you see any odd behavior around locking that provoked this change? A quick find ~/.gradle/caches -name *.lock | wc -l yields 146 different locks :0